configure.ac

```diff
@@ -130,15 +130,18 @@ OPTION_DEFAULT_DISABLE([ugni], [ENABLE_UGNI])
 dnl dnl OPTION_DEFAULT_DISABLE([sos], [ENABLE_SOS])
 dnl dnl OPTION_WITH_OR_BUILD([sos],[../sos],[sos/src sos/include ods/src ods/include])
 
+# PKG_PROG_PKG_CONFIG before all PKG_CHECK_MODULES may or not be called, per pkg.m4.
+m4_ifndef([PKG_CHECK_MODULES],
+      [m4_fatal([pkg.m4 not found. Please install pkg-config (Ubuntu) or pkgconfig (RHEL) package])])
+PKG_PROG_PKG_CONFIG
+
 dnl RDC defaults to no instead of search because amd currently recommends
 dnl multiple non-default path installations (/opt).
 AC_ARG_ENABLE([rdc],
 	[AS_HELP_STRING([--enable-rdc], [attempt components that depend upon AMD rdc tooling @<:@default=no@:>@ for GPUs. Set RDC_CFLAGS=-I$rdcinclude and RDC_LIBS=-L$rdclibdir])],
 	[],
 	[enable_rdc="no"])
 if test "$enable_rdc" != "no"; then
-    m4_ifndef([PKG_CHECK_MODULES],
```

Comment:
@baallan, could you please remove this line from the cray-ugni section that follows?

Reply:
done